packages/design-picker/src/components/blank-canvas-button/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import type { ReactElement } from 'react';
+import './style.scss';
+
+interface Props {
+	className?: string;
+	onSelect: () => void;
+}
+
+const BlankCanvasButton = ( { className, onSelect }: Props ): ReactElement => {
+	const { __ } = useI18n();
+
+	return (
+		<Button className={ classnames( 'blank-canvas-button', className ) } onClick={ onSelect }>
+			{ __( 'Use Blank Canvas' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Blank Canvas' )` *ES Score: 8*